### DISCUSSION TITLE
Timeline on next release?

### DISCUSSION THREAD

**Discussion by tomdotorg** [poblabs/weewx-belchertown#763] [Q&A] [answered]
Is there a target timeline for a 1.3 or 2.0 or whatever is next? I am trying to decide whether to automate just pulling from main here or wait for an official release..

**Comment by poblabs** [comment, marked as answer]
Good question - Issues and pull requests seemed to have calmed down, and I have thought about releasing the master branch as an official release. Not sure how much time I'll have to devote to it in the next couple of days but maybe I just test quick with weewx 4.6.0 and then release it this weekend and hope for the best? 😄

**Comment by tomdotorg** [comment]
That sounds good to me. No rush though.

**Reply by tomdotorg** [reply]
Hey Pat, any update on this? Alternatively, is there a best practice in terms of incorporating development "releases" in the meantime?
